README.md

```diff
@@ -1,96 +1,50 @@
-to configure how the font should be built.
-
-You can create one or more profiles. Each profile will have a set of directives. You can specify alternate glyphs for
-a given ligature. You can also prevent a ligature from being added.
+<img src="./images/operator-mono-lig.png" />
 
-Each profile will be listed with `[profile name]` (the first profile should be named default). The generated font will
-include the profile name. For example `[Go]` would generate the font: _Operator Mono Lig Go_. This way you can configure
-a different set of ligatures for each language in your favorite code editor.
+In addition to the graphic above, the following new ligatures are available:
 
-```ini
-# name of profile (one or more sections, first should be name default)
-[default]
-# glyph=altglyph
-greater_equal.liga=greater_equal.2.liga
+<img src="./images/new-ligatures.png"/>
 
-# do not include glyph (add ! prefix)
-!equal_equal.liga
+# Release notes: 
 
-# another optional section (will create a font named Operator Mono Lig Go)
-[Go]
-... add custom directives for this font ...
-```
+- 🎉 New Version 2.2.5
+  All new redesigned ligatures with better hinting. Updated font generation to support advanced OpenType features
+  like those found in Fira Code. For example, the cursor now moves inside the ligature. It also handles repeating
+  characters properly.
+  
+  <img src="./images/caret-position.gif" />
 
-### New ligatures
+- 🎉 New in Version 2.2
+  With the awesome help from [Mark Skelton](https://github.com/mskelton), we now have the full set of ligatures for
+  the following fonts. Thanks Mark, and thanks to all of you who have been patiently waiting for these ligatures to be completed.
+  - Operator Mono SSm Light/Light Italic
+  - Operator Mono SSm Book/Book Italic
+  - Operator Mono SSm Medium/Medium Italic
+  - Operator Mono SSm Bold/Bold Italic
+  - Operator Mono Light/Light Italic
+  - Operator Mono Book/Book Italic
 
-In addition to the graphic above, the following new ligatures are available:
-<img src="./images/new-ligatures.png"/>
+- 🐛 Fixed in Version 2.2.5
+  Add Operator Mono Book/Book Italic ligatures.
 
-### 🙏 Help Wanted
+- 🐛 Fixed in Version 2.2.4
+  Add some missing ligatures.
 
-As noted above, v2.0 only includes ligatures for **Operator Mono SSm Book**. In order to
-update the other fonts, all the ligatures need to be redone. Unfortunately I just don't
-have the time to do that at the moment. If you have a Mac and are interested in helping,
-please let me know. I will even purchase a license to Gylphs Mini for you.
+- 🐛 Fixed in Version 2.2.3
+  Updated Operator Mono SSm Book Italic ligatures.
 
-You can reach me at kiliman@gmail.com.
+- 🐛 Fixed in Version 2.2.2
+  Fixed glyph widths Operator Mono Light and Light Italic. These fonts accidentally were sized
+  the same as the ScreenSmart versions and caused alignment issues. Also fixed a few ligatures
+  that didn't have the correct weight.
 
-## ☑️ Take the poll
+# ☑️ Take the poll
```

Comment:
I disagree with making this a level 1 header as I would argue that a page should only have a single level 1 header.